_includes/sqlstyle.guide.md

```diff
@@ -277,27 +291,30 @@ FROM my_tmp_table
 
 #### Sub-queries
 
-In PostgreSQL you should probably be doing subqueries with `WITH` clauses.
+In PostgreSQL you should be doing subqueries with `WITH` clauses and avoiding the use of inline
+subqueries.
```

Comment:
Do you have a justification for this other than it's your preference?  I don't think this is always readable for subqueries that have meaningless data.